mem/mem.go

```diff
@@ -50,6 +50,8 @@ type VirtualMemoryStat struct {
 	// https://www.centos.org/docs/5/html/5.1/Deployment_Guide/s2-proc-meminfo.html
 	// https://www.kernel.org/doc/Documentation/filesystems/proc.txt
 	// https://www.kernel.org/doc/Documentation/vm/overcommit-accounting
+	ActiveFile     uint64 `json:"activefile"`
+	InactiveFile   uint64 `json:"inactivefile"`
```

Comment:
I'm against adding these two new fields into the `VirtualMemoryStat` type,  as this type is common to all the platforms gopsutil is compatible with and these values don't make sense on other OSes than Linux.

Reply:
@Lomanic I worried that too, but I couldn't find a way to calculate what I want without these two new fields. I thought `VirtualMemoryStat` struct already had some fields which not compatible with other platforms such as `SReclaimable` and the two new fields value return 0 on other platforms so that this way didn't make a big issue. 

I fully realized that this way isn't graceful at all and will try to investigate getting memavailable without adding these two new fields. Let me know if you have a good idea.

Reply:
@Lomanic I updated this issue. Review, please.

Reply:
Good job on this, I prefer this way and it reflects what has been done in the `process` package with [MemoryInfoExStat](https://godoc.org/github.com/shirou/gopsutil/process#MemoryInfoExStat).